platforms/documentation/docs/src/snippets/java/fixtures/tests/dependencyInsight.out

```diff
@@ -3,7 +3,7 @@
 com.google.code.gson:gson:2.8.5 FAILED
    Failures:
       - Could not resolve com.google.code.gson:gson:2.8.5.
-          - Unable to find a variant providing the requested capability 'com.google.code.gson:gson-test-fixtures':
+          - Unable to find a variant with capabilities matching feature 'test-fixtures':
```

Comment:
💭 "a variant with capabilities matching feature" is slightly awkward wording.

Maybe: "a variant with the requested capabilities and features.  Requested: [ feature 'X', coordinates 'Y']."

Reply:
Are you suggesting always using a list (`[]`) notation here? 

The example you commented on is only a single requested capability selector

Reply:
No, I'm just suggesting changing the words.  I just added the list to show the general form, as is present in the other tests.

Reply:
How would you suggest formatting the single requested selector?

Reply:
Leaving it as one item in a list isn't awful.  Or drop the list and be like:

`"a variant with the requested capabilities and features. Requested: feature 'X'."`

Reply:
I don't feel too strongly about it either, but I do think the change is clearer, especially for someone less familiar to capabilities in general.

Reply:
a variant with the requested capabilities. Requested: feature 'X'.
a variant with the requested capabilities. Requested: coordinates 'X'.
a variant with the requested capabilities. Requested: [feature 'X', coordinates 'Y']